README.md

```diff
@@ -3,15 +3,25 @@
 Welcome to the Personal Travel Assistant project! This application is designed to assist you with your travel needs and make your journey more enjoyable.
 
 ## How to Run
+<ol>
+    <li>
+        <p>
+            Before you begin, make sure you have <a href="https://nodejs.org/en/download/current">Node.js</a> installed on your system.
+        </p>
+    </li>
+    <li>
+        <p>After that, go into the travel-assistant folder</p>
+        <code>cd .\travel-assistant\</code>
+    </li>
+    <li>
+        <p>and then you can set up the project with the following command:</p>
+        <code>npm install</code>
+    </li>
+    <li>
+        <p>You may then run the project with the following command:</p>
```

Comment:
Remove conditional and use affirmative statements e.g. run the project with:

Reply:
will do